packages/suite/src/utils/wallet/coinjoinUtils.ts

```diff
@@ -54,3 +55,28 @@ export const breakdownCoinjoinBalance = ({
 
     return balanceBreakdown;
 };
+
+/**
+ * Transform from coordinator format to coinjoinReducer format `CoinjoinClientFeeRatesMedians`
+ * array => object { name: value-in-vbytes }
+ */
+export const transformFeeRatesMedians = (m: CoinjoinStatusEvent['feeRatesMedians']) => {
+    const [fast, recommended, slow] = m.map(f => f.medianFeeRate);
+    const kb2b = (v: number) => (v ? Math.round(v / 1000) : 1);
```

Comment:
The number is divided, so... is the original unit really kB?

Reply:
accualy its kvbyte (kilo virtual byte) i'll rename it

Reply:
[eda08a3](https://github.com/trezor/trezor-suite/pull/6563/commits/eda08a300f3f85e4a3629dbb47143bcad784a3af)